File: Labwork 4/C#/Part4.cs
using System;

public class Calculator
{
    public static double Calc(double a, double b, char op)
    {
        return op switch
        {
            '+' => a + b,
            '-' => a - b,
            '*' => a * b,
            '/' => a / b,
            _ => throw new ArgumentException("Unsupported operation")
        };
    }

    public static void Main()
    {
        Console.WriteLine(Calc(10, 5, '+'));   
        Console.WriteLine(Calc(10.5, 2, '/')); 

       
    }
}

File: Labwork 4/Python/Part4.py


def calc(a, b, op):
    if op == '+':
        return a + b
    elif op == '-':
        return a - b
    elif op == '*':
        return a * b
    elif op == '/':
        return a / b
    else:
        raise ValueError("Unsupported operation")


print(calc(10, 5, '+'))  
print(calc(10.5, 2, '/'))  


